ironfish/src/blockchain/database/evmReceiptValue.ts

```diff
@@ -0,0 +1,141 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import type { IDatabaseEncoding } from '../../storage/database/types'
+import { Log } from '@ethereumjs/evm'
+import { RunTxResult } from '@ethereumjs/vm'
+import bufio from 'bufio'
+
```

Comment:
nit: why not just call it `EvmReceipt`?

Reply:
I was basing this off of other `*Value` interfaces that we use in databases (`AccountValue`, `AssetValue`, etc.). I think if we end up using this interface outside of just storing data in the database then we should drop the `Value`.